src/PrestaShopBundle/Resources/views/Admin/TwigTemplateForm/bootstrap_3_horizontal_layout.html.twig

```diff
@@ -102,3 +102,12 @@
     {% endif %}
 </div>
 {% endblock text_with_unit_widget %}
+
+{% block ip_address_text_widget %}
+<div class="input-group">
+    {{- block('form_widget_simple') -}}
+    <button type="button" class="btn btn-outline-primary add_ip_button">
+        <i class="material-icons">add_circle</i> Add my IP
```

Comment:
"Add my IP" should be translated, ins't it?

Reply:
Yep, done